lib/fastlane/plugin/wpmreleasetoolkit/actions/common/check_translation_progress.rb

```diff
@@ -0,0 +1,135 @@
+module Fastlane
+  module Actions
+    class CheckTranslationProgressAction < Action
+      def self.run(params)
+        require_relative '../../helper/glotpress_helper.rb'
+
+        UI.message('Check translations status...')
+
+        under_threshold_langs = check_translations(
+                                  glotpress_url: params[:glotpress_url], 
+                                  language_codes: params[:language_codes],
+                                  abort_on_violations: params[:abort_on_violations],
+                                  threshold: params[:min_acceptable_translation_percentage])
+
+        check_results(
+          under_threshold_langs: under_threshold_langs, 
+          threshold: params[:min_acceptable_translation_percentage], 
+          skip_confirm: params[:skip_confirm]
+        ) unless under_threshold_langs.length == 0
+
+        UI.message('Done')
```

Comment:
Nitpick, what do you think of using the `success` method to print a green (?) message to the user?

```suggestion
        UI.success('Done')
```